plugin.video.ahltv/resources/lib/globals.py

```diff
@@ -0,0 +1,97 @@
+import re
+import sys
+import json
+import xbmc, xbmcplugin, xbmcgui, xbmcaddon
+import urllib
```

Comment:
This import is not compatible with Python 3.

Reply:
Updated and tested against Kodi nightly